core/src/main/java/org/mskcc/oncotree/utils/TumorTypesUtil.java

```diff
@@ -284,10 +285,13 @@ private static void setDepthAndTissue(TumorType tumorType, int depth, String tis
                     tumorType.setNCI(crosswalks.get("NCI"));
                 }
                 List<String> umlsIds = new ArrayList<String>();
-                for (String mskConceptId : mskConcept.getConceptIds()) {
-                    umlsIds.add(mskConceptId.replace("MSK", "C"));
+                if (mskConcept.getConceptIds() != null) {
```

Comment:
can we have a concept without an Id?  can't hurt to check for null I guess, but if we do have a null conceptId maybe we should log a warning here (as long as we are checking)